### DISCUSSION TITLE
How do I backup "price configuration" ?

### DISCUSSION THREAD

**Discussion by t-m-cor** [UtilitechAS/amsreader-firmware#818] [Q&A] [answered]
I have tried to search, but have not found any answers. Sorry, if I the answer is elsewhere.

It's time consuming to find and add all tariff's, in Denmark I end up with 11 set of price adjustments.

How do I backup this ? I have tried to use ->(I)->"Backup & Restore", but I do not see my "price configuration"'s in the downloaded .cfg file, no matter what options I choose.

It would be nice to have a .cfg to share, as most of the "price configuration" is common for all in my country (Denmark).

Greetings from Denmark,
Thomas

**Comment by ArnieO** [comment, marked as answer]
Hi Thomas,
I think this is a good recommendation for an enhancement. 
In some cases, the price configuration can indeed become quite complex. 
Thank you for the suggestion, we note it for a future firmware update!
(ping @gskjold )

**Comment by gskjold** [comment]
Implemented
